examples/jaeger/src/main/java/io/opentelemetry/example/JaegerExample.java

```diff
@@ -0,0 +1,82 @@
+package io.opentelemetry.example;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.opentelemetry.exporters.jaeger.JaegerGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.TracerSdkFactory;
+import io.opentelemetry.sdk.trace.export.SimpleSpansProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
+import io.opentelemetry.trace.TracerFactory;
+
+public class JaegerExample {
+  // Jaeger Endpoint URL and PORT
+  private String ip; // = "jaeger";
+  private int port; // = 14250;
+  // OTel API
+  private Tracer tracer;
+  private TracerSdkFactory tracerFactory;
+  // Create a channel towards Jaeger end point
+  private ManagedChannel jaegerChannel;
+  // Export traces to Jaeger
+  private JaegerGrpcSpanExporter jaegerExporter;
+
+  public JaegerExample(String ip, int port) {
+    this.ip = ip;
+    this.port = port;
+    // Instantiate the OTel factory
+    this.tracerFactory = OpenTelemetrySdk.getTracerFactory();
+    // Give a name to the tracer
+    this.tracer = tracerFactory.get("io.opentelemetry.example.JaegerExample");
```

Comment:
This should be gotten via the API:
`OpenTelemetry.getTracerFactory().get(...);`